ts/packages/cli/src/helpers/synchronizers/logic/Vercel.ts

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import _ from 'lodash';
+import { EvaluatedConfigs } from '@configu/ts';
+import { CONFIG_SYNCHRONIZER_LABEL } from '@configu/lib';
+
+const label = CONFIG_SYNCHRONIZER_LABEL.Vercel;
+// * `target` and `type` are mandatory for the "Create one or more environment variables" Vercel endpoint.
+// * Endpoint doc: https://vercel.com/docs/rest-api#endpoints/projects/create-one-or-more-environment-variables
+const DEFAULT_TARGETS = ['development', 'preview', 'production'];
+const DEFAULT_TYPE = 'encrypted';
+
+export type VercelConfiguration = {
+  // * Vercel docs: https://devcenter.heroku.com/articles/platform-api-reference#config-vars
+  token: string;
+  appUid: string;
+};
+type VercelEnvVar = { key: string; id: string; [otherKeys: string]: unknown };
+
+// TODO - Decide between syncVercelConfigs (previous name) and syncVercelEnvVars
+export const syncVercelEnvVars = async ({
```

Comment:
@rannn505 We previously named this `syncVercelConfigs`, Vercel names their 'configs' as 'environment variable' so I thought `syncVercelEnvVars` would be more appropriate here. Opinion?

Reply:
We agreed on `syncVercelEnvironmentVariables`.